### PR TITLE
KFSTI-1754

### PR DESCRIPTION
**Pull request by smith750**
Move work directory structure for now

### CODE REVIEW

File: src/main/config/work/staging/tem/error/placeholder.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+test
```
Comment (bhutchinson) [inline src/main/config/work/staging/tem/error/placeholder.txt]: I'm wondering about why this line was added?

File: src/main/config/work/staging/sys/batchContainer/placeholder.txt [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  Adieu! the fancy cannot cheat so well
+    As she is fam'd to do, deceiving elf.
+  Adieu! adieu! thy plaintive anthem fades
+    Past the near meadows, over the still stream,
+      Up the hill-side; and now 'tis buried deep
+          In the next valley-glades:
+    Was it a vision, or a waking dream?
+      Fled is that music:--Do I wake or sleep?                 80
+
+-- John Keats
```
Comment (bhutchinson) [inline src/main/config/work/staging/sys/batchContainer/placeholder.txt]: easter egg

File: src/main/config/work/staging/tem/error/placeholder.txt (hunk 1 above)
Comment (smith750) [inline src/main/config/work/staging/tem/error/placeholder.txt]: Yeah, I didn't do that one, so I'm unsure myself.  I can get rid of those.

File: src/main/config/work/staging/sys/batchContainer/placeholder.txt (hunk 2 above)
Comment (bhutchinson) [inline src/main/config/work/staging/sys/batchContainer/placeholder.txt]: so, I'm just noticing this - it looks like these files were added to the new location, but not moved/deleted from the old location? am I misreading?

Comment (bhutchinson) [inline src/main/config/work/staging/sys/batchContainer/placeholder.txt]: add since KFSTI-1682 is still pending

**Pull request merged**